FalmouthFalcons/iteration-3
# Shopping Cart Now Has User's Total Price

Description:
-Add description about what this pull request covers.
Shopping Cart Now Has User's Total Price
Number of Fixes:
-Type of number of fixes included in this PR.

Related Ticket(s):
- List related ticket(s) that apply to the work done in this pull request

Problem to Solve:
- Describe the problem(s) this pull request solves.

Proposed Changes:
-Describe the proposed changes.

Expected Behavior:
- Describe the expected behavior.

Steps to Test Solution
1)git fetch
2)git pull origin CM-total
3)test to see total is on shopping cart page

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
